tests/classes/test_hypergraph.py

```diff
@@ -267,3 +271,143 @@ def test_add_node_to_edge():
         "fruits": ["apple", "banana", "pear"],
         "veggies": ["lettuce"],
     }
+
+
+def test_add_edges_from_format1():
+    edges = [[0, 1], [1, 2], [2, 3, 4]]
+    H = xgi.Hypergraph()
+    H.add_edges_from(edges)
+    assert list(H.edges.members()) == edges
+
+    H1 = xgi.Hypergraph(edges)
+    H = xgi.Hypergraph(H1.edges)
+    H.add_edges_from(edges)
+    assert list(H.edges.members()) == edges
+
+    edges = {frozenset([0, 1]), frozenset([1, 2]), frozenset([2, 3, 4])}
+    H = xgi.Hypergraph()
+    H.add_edges_from(edges)
+    assert list(H.edges.members()) == [list(e) for e in edges]
+
+    edges = [[0, 1], frozenset([1, 2]), (2, 3, 4)]
+    H = xgi.Hypergraph()
+    H.add_edges_from(edges)
+    assert list(H.edges.members()) == [list(e) for e in edges]
+
+    edges = [["foo", "bar"], ["bar", "baz"], ["foo", "bar", "baz"]]
+    H = xgi.Hypergraph()
+    H.add_edges_from(edges)
+    assert list(H.nodes) == ["foo", "bar", "baz"]
+    assert list(H.edges.members()) == [list(e) for e in edges]
+
+    edges = [["a", "b"], ["b", "c"], ["c", "d", "e"]]
```

Comment:
is this not covered by the foor bar baz case above?

Reply:
I had to add this one because some of the formats accepted by `add_edges_from` differentiate between some of the elements having three elements (covered by "foo" "bar" "baz") and one element (covered by this test).